core/src/main/java/org/apache/calcite/runtime/Resources.java

```diff
@@ -1,10 +1,10 @@
 /*
- * Licensed to Julian Hyde under one or more contributor license
```

Comment:
@julianhyde , AFAIK, we must not retain the original copyright owners in the source file, rather we move them to the NOTICE file (see https://www.apache.org/legal/src-headers.html#headers )

Reply:
A few lines down it says "Do not modify or remove any copyright notices or licenses within third-party works.".

Reply:
Thanks for the pointer, and I kept the headers. I do not like how "source-included" file complicates build/edit operations as different files might happen to have different code styles, headers, etc, etc.